composer.json

```diff
@@ -16,33 +16,30 @@
         }
     ],
     "require": {
-        "php": "~5.5|~7.0",
-        "sonata-project/admin-bundle": "~3.0",
-        "sonata-project/block-bundle": "^2.3.2",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/property-access": "~2.2|~3.0",
-        "doctrine/phpcr-odm": "~1.1",
-        "doctrine/phpcr-bundle": "~1.1",
-        "symfony-cmf/tree-browser-bundle": "~2.0",
-        "symfony-cmf/resource-rest-bundle": "~1.0"
+        "php": "^5.5 || ^7.0",
```

Comment:
> instead of the quite common `|`?

This is the problem: it shouldn't.

`||` is the new standard. See examples of docs: https://getcomposer.org/doc/04-schema.md